resource/wrapper/retryresource/wrap.go

```diff
@@ -31,10 +30,9 @@ func Wrap(resources []resource.Interface, config WrapConfig) ([]resource.Interfa
 
 	for _, r := range resources {
 		c := Config{
+			BackOff:  config.BackOffFactory(),
```

Comment:
The backoff factory here is defaulted at the wrapping stage and this is what we only use in the operators.